### PR TITLE
Add user guide for User Onboarding

### PR DESCRIPTION
**Pull request by Wei-Xinn**
# Changes 
This PR updates only the User Guide related to user onboarding
A staging version of the updated User Guide can be found [here](https://wei-xinn.github.io/tp/UserGuide.html).

# Closes Issues
* #72

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -72,6 +72,14 @@ Shows a message explaning how to access the help page.
 
 Format: `help`
 
+### Onboarding guide
+The user will be prompted to take a quick tour of the ContacX's basic functions via a series of step-by-step instructions on the first run of the program.
```
Comment (sharpstorm) [inline docs/UserGuide.md:76]: Should a friendlier tone be used? E.g.
> When you run the program for the first time, you will be prompted to ...

Not a must, but I feel it better fits the rest of the user guide

Comment (Wei-Xinn) [reply docs/UserGuide.md:76]: Good suggestion! Will update accordingly.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -73,8 +73,8 @@ Shows a message explaning how to access the help page.
 Format: `help`
 
 ### Onboarding guide
-The user will be prompted to take a quick tour of the ContacX's basic functions via a series of step-by-step instructions on the first run of the program.
- 
+When you run the program for the first time, you will be prompted to take a quick tour of the ContacX's basic functions guided through a series of step-by-step instructions.
```
Comment (sharpstorm) [inline docs/UserGuide.md:76]: Spelling error here, please fix.
```suggestion
When you run the program for the first time, you will be prompted to take a quick tour of the ContaX's basic functions guided through a series of step-by-step instructions.
```

**Review by sharpstorm — APPROVED** [review]
lgtm.

**Review by dandaandaaaaaan — APPROVED** [review]
lgtm, when u actually implement remember to have some button or visual cue so that the user will know how to dismiss the notices, and save some form of state such that it will only be shown on first run

**Pull request merged**